<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicio_juego_cargar</class>
 <widget class="QWidget" name="inicio_juego_cargar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_clave">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cargar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cargar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_usuario">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
